<ui version="4.0" >
 <class>TabSearch</class>
 <widget class="QWidget" name="TabSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Type in word(s) to search for:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblHelp" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&lt;url href="help">Help&lt;/url></string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="searchBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="maxCount" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGo" >
       <property name="text" >
        <string>&amp;Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table" >
     <column>
      <property name="text" >
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Location</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
